lib/active_record/connection_adapters/postgis/create_connection.rb

```diff
@@ -35,8 +33,12 @@ def postgis_connection(config)
         valid_conn_param_keys = PG::Connection.conndefaults_hash.keys + [:requiressl]
         conn_params.slice!(*valid_conn_param_keys)
 
-        conn = PG.connect(conn_params)
-        ConnectionAdapters::PostGISAdapter.new(conn, logger, conn_params, config)
+        ConnectionAdapters::PostGISAdapter.new(
+          ConnectionAdapters::PostGISAdapter.new_client(conn_params),
```

Comment:
This should be done in a fourth commit. Changes has been made to keep uniformity with AR61's `PostgreSQLAdapter`

Reply:
It's the same as rails/rails#37291

Reply:
Found it. We need another change

rails/rails#40110